tests/unit-tests/test-class-admin.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+class Sensei_Class_Admin_Test extends WP_UnitTestCase {
+
+	/**
+	 * Constructor function
+	 */
+	public function __construct() { // phpcs:ignore Generic.CodeAnalysis.UselessOverridingMethod.Found
+		parent::__construct();
+	}
+
+	/**
+	 * Setup function
+	 *
+	 * This function sets up the lessons, quizes and their questions. This function runs before
+	 * every single test in this class
+	 *
+	 * @return void
+	 */
+	public function setup() {
+		parent::setup();
+
+		$this->factory = new Sensei_Factory();
+	}//end setup()
+
+	public function tearDown() {
+		parent::tearDown();
+		$this->factory->tearDown();
+	}
+
+	/**
+	 * Testing the admin class to make sure it is loaded
+	 *
+	 * @return void
+	 */
+	public function testClassInstance() {
+		// test if the class exists
+		$this->assertTrue( class_exists( 'WooThemes_Sensei_Admin' ), 'Sensei Admin class does not exist' );
+	} // end testClassInstance
+
+	/**
+	 * Setup function for duplicate course
+	 *
+	 * This function mocks the redirect method, create and set the user, create a course
+	 * with some lessons and set the needed params.
+	 *
+	 * @param $qty_lessons How many lessons to create.
+	 * @return array       Created course and list of lessons id.
+	 */
+	private function duplicate_course_with_lessons_setup( $qty_lessons ) {
```

Comment:
Conventionally, we would add private helpers like this to the end of the test class.

Reply:
Perfect! Moved to the end of the class ;)